query_execution/WorkOrdersContainer.hpp

```diff
@@ -220,8 +226,9 @@ class WorkOrdersContainer {
    * @param operator_index The index of the operator in the query DAG.
    **/
   void addNormalWorkOrder(WorkOrder *workorder, const std::size_t operator_index) {
-    DEBUG_ASSERT(workorder != nullptr);
-    DEBUG_ASSERT(operator_index < num_operators_);
+    DCHECK(workorder != nullptr);
+    DCHECK(operator_index < num_operators_);
+    workorder->setQueryID(query_id_);
```

Comment:
I was wondering if we could set `query_id` for a `WorkOrder` during its construction.